extension/android/jni/jni_layer.cpp

```diff
@@ -243,17 +243,29 @@ class ExecuTorchJni : public facebook::jni::HybridClass<ExecuTorchJni> {
       facebook::jni::alias_ref<jstring> modelPath,
       facebook::jni::alias_ref<
           facebook::jni::JMap<facebook::jni::JString, facebook::jni::JString>>
-          extraFiles) {
-    return makeCxxInstance(modelPath, extraFiles);
+          extraFiles,
+      jint loadMode) {
+    return makeCxxInstance(modelPath, extraFiles, loadMode);
   }
 
   ExecuTorchJni(
       facebook::jni::alias_ref<jstring> modelPath,
       facebook::jni::alias_ref<
           facebook::jni::JMap<facebook::jni::JString, facebook::jni::JString>>
-          extraFiles) {
-    module_ = std::make_unique<Module>(
-        modelPath->toStdString(), Module::LoadMode::Mmap);
+          extraFiles,
+      jint loadMode) {
```

Comment:
Can we use `Module::LoadMode` for `loadMode` here instead of `jint`

Reply:
This function call is from Java. It's very hard to expose `Module::LoadMode` to Java. It needs additional binding work. Instead we copy https://github.com/pytorch/executorch/pull/4578/files#diff-8b3b129023d02e6f4a306566ef9859d103066bb614bb501349960e9e3fce5270R18-R29 and match  https://github.com/pytorch/executorch/pull/4578/files#diff-12aef13b6cef99a96c007a8da411b1bd68d1ec11e1b6246745b319cae1f1bba3R258-R266 manually. That's a common practice.